hadoop-ozone/dev-support/checks/_mvn_unit_report.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## generate summary txt file
+find "." -name 'TEST*.xml' -print0 \
+    | xargs -n1 -0 "grep" -l -E "<failure|<error" \
+    | awk -F/ '{sub("'"TEST-JUNIT_TEST_OUTPUT_DIR"'",""); sub(".xml",""); print $NF}' \
```

Comment:
Is `JUNIT_TEST_OUTPUT_DIR` part of the filename in some environment?  In PR runs it is not, hence, `TEST-` [prefix is retained](https://github.com/elek/ozone-ci/blob/master/pr/pr-hdds-1950-pfqf7/integration/summary.txt):

```
TEST-org.apache.hadoop.ozone.client.rpc.TestOzoneRpcClient
TEST-org.apache.hadoop.ozone.client.rpc.TestOzoneRpcClientWithRatis
TEST-org.apache.hadoop.ozone.client.rpc.TestSecureOzoneRpcClient
TEST-org.apache.hadoop.ozone.client.rpc.TestOzoneAtRestEncryption
TEST-org.apache.hadoop.ozone.scm.node.TestQueryNode
```

```suggestion
    | awk -F/ '{sub("'"TEST-"'",""); sub(".xml",""); print $NF}' \
```